grpc/proto/v1/generative.proto

```diff
@@ -20,7 +20,7 @@ message GenerativeSearch {
     string task = 1;
     optional TextArray properties = 2;
     // only allow one at the beginning, but multiple in the future
-    repeated GenerativeProvider queries = 3;  
+    repeated GenerativeProvider queries = 3;
```

Comment:
aren't we missing more changes to this file? I'm looking for this `primitive properites` change, where is it?

Reply:
Ah yes, I decided against that change as it would require some non-trivial deprecated code, e.g. a `uses_x` boolean and then some functions for supported/deprecated logic. I decided against introducing that in this PR since it would make the changes bigger for small material benefit